src/Bridge/Doctrine/Orm/SubresourceDataProvider.php

```diff
@@ -198,16 +200,42 @@ private function buildQuery(array $identifiers, array $context, QueryNameGenerat
                 ->from($identifierResourceClass, $alias);
         }
 
+        $lastIdentifer = $remainingIdentifiers === 1;
+        $association = $classMetadata->hasAssociation($previousAssociationProperty) ? $classMetadata->getAssociationMapping($previousAssociationProperty) : [];
+        $optimizable = $lastIdentifer && (
```

Comment:
Better explain *why* it's optimizable and make that the variable name here.

Reply:
I added a long comment about the optimization, when using `$optimizable`. I could not find a variable name that was both concise and explanatory though.

Is that ok with you?

Reply:
`$needsSubquery`? (So it'd have the opposite value.)

Reply:
Or `$isLeaf` to make it merely descriptive and decoupled from the implementation.

Reply:
I reworked the way the code is written, to have more readable code branches and variable name. WDYT?

Reply:
`$eligibleOneToOneOptimization` -> `$oneToOneBidirectional`
`$eligibleOneToManyOptimization` -> `$oneToManyBidirectional`

Same idea... The names can be merely descriptive.

Reply:
Done :)